### DISCUSSION TITLE
p/Nginx%E8%AF%A6%E8%A7%A3/

### DISCUSSION THREAD

**Discussion by giscus** [baixiaozhou/hexo-blog#5] [Announcements]
# p/Nginx%E8%AF%A6%E8%A7%A3/

Nginx 使用介绍

https://baixiaozhou.github.io/p/Nginx%E8%AF%A6%E8%A7%A3/

**Comment by panyy97** [comment]
nginx-mod-http-perl
nginx-mod-http-image-filter
nginx-mod-stream-filter
nginx-mod-stream
nginx-mod-mail
nginx-mod-http-xslt-filter
nginx-all-modules
nginx-filesystem
nginx
楼主楼主，nginx相关rpm包一般有这些，除了核心的nginx包之外，其他的一般用于什么功能，或者提供了哪些能力，蹲后续
